utility/servicer/module.go

```diff
@@ -192,9 +192,9 @@ func (s *servicer) executeRelay(relay *coreTypes.Relay) (*coreTypes.RelayRespons
 	case *coreTypes.Relay_JsonRpcPayload:
 		return s.executeHTTPRelay(relay.Meta, payload.JsonRpcPayload)
 	case *coreTypes.Relay_RestPayload:
-		return nil, fmt.Errorf("Error executing relay on application %s: REST not supported", relay.Meta.ApplicationAddress)
+		return s.executeRESTRelay(relay.Meta, payload.RestPayload)
```

Comment:
My last NIT on this PR is just the naming and comments of these functions.

In the switch statement, we have:
`case *coreTypes.Relay_JsonRpcPayload:` -> `s.executeHTTPRelay(`
`case *coreTypes.Relay_RestPayload` -> `s.executeRESTRelay`

The first header is:

```go
// executeHTTPRequest performs the HTTP request that sends the relay to the chain's/service's URL.
func (s *servicer) executeHTTPRelay(meta *coreTypes.RelayMeta, payload *coreTypes.JSONRPCPayload) (*coreTypes.RelayResponse, error) {
```

The second header is:

```go
// INCOMPLETE(#860): RESTful service relays: basic checks and execution through HTTP calls
func (s *servicer) executeRESTRelay(meta *coreTypes.RelayMeta, _ *coreTypes.RESTPayload) (*coreTypes.RelayResponse, error) {
```

My suggestion is this:

```go
// executeJsonRPCRelay performs the relay for JSON-RPC payloads, sending them to the chain's/service's URL.
func (s *servicer) executeJsonRPCRelay(meta *coreTypes.RelayMeta, payload *coreTypes.JSONRPCPayload) (*coreTypes.RelayResponse, error) {
```

```go
// executeRESTRelay performs the relay for REST payloads, sending them to the chain's/service's URL.
// INCOMPLETE(#860): RESTful service relays: basic checks and execution through HTTP calls.
func (s *servicer) executeRESTRelay(meta *coreTypes.RelayMeta, _ *coreTypes.RESTPayload) (*coreTypes.RelayResponse, error) {
```

If the REST (a design pattern) payload needs to use `httpRelay` under the hood, that's fine.
If the JSON RPC (a specific protocol) leverages `httpRelay` under the hood (but could theoretically operate over gRPC or a weboscket if we wanted it to), that's fine.

Wdyt?

Reply:
Thank you for the review. Updated the comments and the function names.